pages/products.js

```diff
@@ -0,0 +1,17 @@
+import Products from '../components/organism/Products/Products'
+
+function ProductsIndex({pageProps}) {
+  return (
+    <Products {...pageProps}/>
```

Comment:
Can you please move the logic of pages to a components inside templates folder as this would complete the atomic design structure.